### PR TITLE
cpio: added no-absolute-filenames

### PR DESCRIPTION
**Pull request by mlfm**
I couldn't translate for pages.ko/common/cpio.md.

- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by mlfm** [comment]
I do not want to sign the CLA. 
Claim ownership and make a new pull request if needed.

File: pages.ko/common/cpio.md [hunk 1]
```diff
@@ -13,4 +13,4 @@
 
 - 아카이브에 모든 파일을 [i]선택하여 필요한 경우 [v]상세모드로 [d]디렉토리를 생성:
```
Comment (zdroid) [inline pages.ko/common/cpio.md]: This needs an update too. @Proscream maybe you can help?

Comment (mlfm) [reply pages.ko/common/cpio.md]: Google translate:
아카이브에서 모든 파일을 [i]선택하여 현재 디렉토리를 기준으로 필요한 경우 자세한 모드로 디렉토리를 생성하십시오
Could not find the place to put [v],[d] and proper spacing.

**Comment by sbrl** [comment]
Hey there, @mlfm!

Thanks for the contribution. Unfortunately, we are unable to accept it unless you were to sign the CLA. The CLA is a small agreement that ensures that all licensing concerns are taken care of. For reference, by clicking the "Contributor Licence Agreement" hyperlink above in @CLAassistant's comment, you can see our CLA:

```markdown
# TDLR Pages contributor agreement

1. I assert that any past, present and future contributions I make to the tldr-pages project are my own work, and that I have the right to license them.
2. I agree to license all my contributions to the tldr-pages project under the terms of the license included in the tldr-pages repository I’m contributing to.
3. I grant the tldr-pages organization a license to distribute my contributions under any open content license compliant with the Open Definition [a], or any open source software license approved by the Open Source Initiative [b], as applicable.

a. http://opendefinition.org/licenses
b. https://opensource.org/licenses/alphabetical
```

@pxgamer / @agnivade / @mebeim, could I word this better?

**Comment by mebeim** [comment]
@sbrl I think the wording is fine. If @mlfm really doesn't want to sign the CLA someone else should take "ownership" of this PR by either (A) closing this one and copying the work to another one or (B) force-pushing on the forked branch changing the commit author. 

Moreover:

1. I think this modification warrants a new example, as he one that is being modified becomes a little bit too intricate. The page only has 3 examples so a third one is fine.
2. There's no need to update the Korean translation if we don't have a good translation, Google Translate is definitely bad at translating such complicated languages.

**Comment by mlfm** [comment]
Just a reminder that, I made the fix mebeim wanted.
This pull request is ready to be CLAIM by anyone.

**Comment by sbrl** [comment]
Thanks, @mebeim. Unfortunately, I'm not sure we could legally claim this PR and change the authorship information without adequately covering all our bases.

It's a huge shame, but if you are unwilling to sign the CLA I'm afraid we will have to close this PR, @mlfm :-(

**Comment by mlfm** [comment]
I cannot sign the CLA, I need to keep my email private.
So I am closing this. Just to state it again. I give this code freely, pillage it and do as you like.
Keep up the good work with tldr and thanks for all the fish.

**Comment by sbrl** [comment]
@mlfm No problem. We absolutely respect your decision :-)

**Pull request closed without merging**